src/index.html

```diff
@@ -17,11 +17,481 @@
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="index.html"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!--#endregion Header-->
+    <main class="catalog">
+      <!--#region CARD-->
+
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__content">
+          <div class="card__image"></div>
+
+          <div class="card__title">
+            <p class="card__title-text">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+          </div>
+
+          <div class="card__code">
+            <p class="card__code-text">Product code: 195434</p>
+          </div>
+
+          <div class="card__rating">
+            <div class="card__rating-stars">
+              <div class="card__rating-star"></div>
+              <div class="card__rating-star"></div>
+              <div class="card__rating-star"></div>
+              <div class="card__rating-star"></div>
+              <iv class="card__rating-star"></iv>
+            </div>
+            <p class="card__rating-text">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+
+          <div class="card__buy-wrapper">
+            <a
+              href="#"
+              class="card__buy"
+              data-qa="card-hover"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+      </div>
```

Comment:
It seems like your card component is repeating multiple times. It would be better to use a loop to generate these components to avoid repeating code. In the context of HTML, you can't do it directly, but if you are using a templating engine or a frontend framework like React, Angular or Vue.js, you can easily loop through an array to generate repeated components.